app/routes/submit.js

```diff
@@ -0,0 +1,52 @@
+const { MessageSender } = require('ffc-messaging')
+const { GET, POST } = require('../constants/http-verbs')
+const { USER } = require('ffc-auth/scopes')
+
+module.exports = [
+  {
+    method: GET,
+    path: '/submit',
+    options: {
+      auth: { strategy: 'jwt', scope: [USER] }
+    },
+    handler: async (_request, h) => {
+      return h.view('submit')
+    }
+  },
+  {
+    method: POST,
+    path: '/submit',
+    options: {
+      auth: false
+    },
```

Comment:
Why no authentication on POST?

Reply:
No solid reasoning for this as originally was going off the format of `POST` on the `sign-in` and `sign-in-oidc` routes, but realising now that these two differ from the submit route in function/purpose so I've included the authentication strategy back in.